Dockerfile.devel

```diff
@@ -56,14 +56,18 @@ RUN chmod +x /app/launch.sh
 COPY ./deploy/entryPoint.sh /app/entryPoint.sh
 RUN chmod +x /app/entryPoint.sh
 
-COPY ./rcsb/app/tests-file/test-data/config/example-data.cif    /app/CACHE/config/example-data.cif
-RUN chown ubuntu /app/CACHE
+COPY ./rcsb/app/tests-file/test-data/config/example-data.cif /app/CACHE/config/example-data.cif
+COPY ./rcsb/app/tests-file/test-output/CACHE/config/example-config-2.json /app/CACHE/config/example-config.json
```

Comment:
@connor-parker-rcsb , This is a little problematic, since if a user clones the repository from scratch, they won't have this file in their CACHE. Also I would stray away from using the `example-config-2.json` filename if possible. Are you able resolve this line to something more standardized and not dependent on the CACHE?